File: accudrop/app/src/main/java/me/chrislane/accudrop/db/converter/UuidConverter.java
package me.chrislane.accudrop.db.converter;

import android.arch.persistence.room.TypeConverter;

import java.util.UUID;

public class UuidConverter {
    @TypeConverter
    public static UUID toUuid(String uuidString) {
        return UUID.fromString(uuidString);
    }

    @TypeConverter
    public static String toStr(UUID uuid) {
        return uuid.toString();
    }
}

File: app/src/main/java/me/chrislane/accudrop/db/converter/UuidConverter.kt
package me.chrislane.accudrop.db.converter

import androidx.room.TypeConverter
import java.util.*

class UuidConverter {

    @TypeConverter
    fun toUuid(uuidString: String): UUID {
        return UUID.fromString(uuidString)
    }

    @TypeConverter
    fun toStr(uuid: UUID): String {
        return uuid.toString()
    }
}
